### ISSUE TITLE
psbt.sign not support pubkey type witness_v1_taproot

### ISSUE THREAD

**Issue by flex-enjoy** [Simplexum/python-bitcointx#78]
For example: 

Bitcoin Testnet address `tb1ppsdp29zhvd0m7rmvn07sqv53eesjllq2w9lk97vy9qck3y37hudsv8kvfp` cannot pass the psbt.sign

I checked the code https://github.com/Simplexum/python-bitcointx/blob/master/bitcointx/core/psbt.py#L1027  , seems witness only support v0 keyhash & v0 scripthash

**Comment by dgpv** [comment]
Yes, `core/psbt.py` does not support taproot - even psbt fields for taproot are not there. That would be quite a bit of work to add this support.
